### PR TITLE
S2 Nbart Product Definition

### PR DESCRIPTION
**Pull request by kieranricardo**
This PR adds a product definition for the S2 Nbart data that will soon be added to AWS. The metadata type is `eo3` and the product names are `s2a_ard_granule` and `s2b_ard_granule`. I tested this locally and successfully indexed some datasets stored here 

This product definition was adapted from https://github.com/GeoscienceAustralia/dea-config/blob/master/products/ga_s2_ard_nbar/ga_s2_ard_nbar_granule.yaml and the exact changes can be seen by comparing the first and last commits. tldr I changed `nbar` to `nbart` and changed the `metadata` field to just contain the product name, but I'm not sure whether the deleted `metadata` was important!

### CODE REVIEW

File: digitalearthau/config/eo3/products/s2_ard_granule.yaml [hunk 1]
```diff
@@ -0,0 +1,697 @@
...
+            2281, 2282, 2283, 2284, 2285, 2286, 2287, 2288, 2289, 2290, 2291, 2292,
+            2293, 2294, 2295, 2296, 2297, 2298, 2299, 2300, 2301, 2302, 2303, 2304,
+            2305, 2306, 2307, 2308, 2309, 2310, 2311, 2312, 2313, 2314, 2315, 2316,
+            2317, 2318, 2319, 2320]
+    units: '1'
+metadata:
+    product:
+        name: s2a_ard_granule
+metadata_type: eo3
+name: s2a_ard_granule
```
Comment (alexgleith) [inline digitalearthau/config/eo3/products/s2_ard_granule.yaml]: Can we please move these up to the top

File: digitalearthau/config/eo3/products/s2_ard_granule.yaml [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+            2256, 2257, 2258, 2259, 2260, 2261, 2262, 2263, 2264, 2265, 2266, 2267,
+            2268, 2269, 2270, 2271, 2272, 2273, 2274, 2275, 2276, 2277, 2278, 2279,
+            2280, 2281, 2282, 2283, 2284, 2285, 2286, 2287, 2288, 2289, 2290, 2291,
+            2292, 2293, 2294, 2295, 2296, 2297, 2298, 2299, 2300, 2301, 2302, 2303]
+    units: '1'
+metadata:
+    product:
+        name: s2b_ard_granule
+metadata_type: eo3
+name: s2b_ard_granule
```
Comment (alexgleith) [inline digitalearthau/config/eo3/products/s2_ard_granule.yaml]: These up top of this product too

**Review by alexgleith — CHANGES_REQUESTED** [review]
These products are for AWS, so need to be here: https://github.com/GeoscienceAustralia/digitalearthau/tree/develop/digitalearthau/config/eo3/products-aws

And can we please include the license section, have a look at the WO or FC docs for an example.

**Comment by kieranricardo** [comment]
@alexgleith thanks for the speedy review! I moved it into the right folder, and added the same license as the other docs

**Review by alexgleith — APPROVED** [review]

**Pull request merged**